.github/actions/next-stats-action/src/index.js

```diff
@@ -132,6 +132,9 @@ if (!allowedActions.has(actionInfo.actionName) && !actionInfo.isRelease) {
         )
         .catch(console.error)
 
+      console.log(await exec(`ls ${path.join(__dirname, '../native')}`))
+      console.log(await exec(`cd ${dir} && ls ${dir}/packages/next-swc/native`))
```

Comment:
You could probably pass a `cwd` option here instead of `cd` right?

Reply:
Oh `cd` isn't even needed here was just copied from other logs